google-http-client/src/main/java/com/google/api/client/http/HttpTransport.java

```diff
@@ -158,4 +158,13 @@ public boolean isMtls() {
    * @since 1.4
    */
   public void shutdown() throws IOException {}
+
+  /**
+   * Default implementation of whether the http transport is shutdown or not.
+   *
+   * @since 1.4
```

Comment:
This version should be latest.

Reply:
Done, changed to 1.44.0 (the next release).